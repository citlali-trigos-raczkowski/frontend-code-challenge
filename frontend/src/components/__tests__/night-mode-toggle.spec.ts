import { shallowMount, createLocalVue } from "@vue/test-utils";
import NightModeToggle from "../night-mode-toggle.vue";
import VueRouter from "vue-router";
import { NightModeToggleMockData } from "../../../tests/mocks/mock-props";

const localVue = createLocalVue();
localVue.use(VueRouter);

describe("component/night-mode-toggle.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(NightModeToggle, {
      propsData: NightModeToggleMockData,
      localVue,
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
