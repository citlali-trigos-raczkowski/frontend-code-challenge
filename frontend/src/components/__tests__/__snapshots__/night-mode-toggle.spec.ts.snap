// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/night-mode-toggle.vue renders correctly 1`] = `
<div
  id="night-mode"
>
  <v-layout-stub
    tag="div"
  >
    <v-spacer-stub />
     
    <v-switch-stub
      backgroundcolor=""
      color="yellow"
      errorcount="1"
      errormessages=""
      id="night-mode-toggle"
      inset="true"
      messages=""
      ripple="true"
      rules=""
      successmessages=""
      valuecomparator="[Function]"
    />
  </v-layout-stub>
</div>
`;
